app/assets/stylesheets/application.scss

```diff
@@ -18,10 +18,42 @@
 
 body {
   background-color: #e0e0e0;
+  background-image: url("/assets/woody.jpg");
+  // background-repeat: no-repeat;
+  position: relative;
+  margin-top: 35px;
+  margin-bottom: 35px;
+  h1 {
+    text-align: center;
+  }
+
+
 }
 
 .container-fluid {
   max-width: 80%;
+  background-color: transparentize(white, .3);
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`.3` should be written with a leading zero as `0.3`